src/index.template.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
added this index template here, so it rebuilds on every `npm run build` since the css and js file names will be unique on every build (they have a hash added to the name)